### ISSUE TITLE
PGA not working?

### ISSUE THREAD

**Issue by zag2me** [jacoduplessis/golf#7]
@jacoduplessis I just randomly browsed here looking at why my PGA tour livescore importer stopped working on my site (www.thesportsdb.com). 

Can you also confirm it no longer works puiblicly?

https://github.com/jacoduplessis/golf/blob/78791935ff4c98641652e0688789a0b316d54a91/pga/pga.go#L56

Its a shame as it was a nice API :)

**Comment by jacoduplessis** [comment]
Yeah it's not working at the moment. Planning to fix it some time this week.

**Comment by zag2me** [comment]
Awesome, but I think the API is dead for public use.

**Comment by jacoduplessis** [comment]
Seems they have added some user tokens with expiry. Nothing insurmountable.

**Comment by jacoduplessis** [comment]
I have created a new golf data API available here: https://pygolf.jacoduplessis.co.za/api

The data format is a bit different.

Planning to archive this project and put the new code online soon.
